packages/features/ee/workflows/components/WorkflowStepContainer.tsx

```diff
@@ -515,6 +549,29 @@ export default function WorkflowStepContainer(props: WorkflowStepProps) {
                   )}
                 </>
               )}
+              {form.getValues(`steps.${step.stepNumber - 1}.action`) !== WorkflowActions.SMS_ATTENDEE && (
```

Comment:
We don't allow testing the action ‘send SMS to attendee’ as we do not want to send SMS to random attendees from a booking. If a user wants to see how the SMS template looks like, they need to switch to ‘Send SMS to specific number`